frontend/resources/run_test.py

```diff
@@ -17,18 +17,17 @@ def run_command(command, log_file):
                 print(f"Command '{command}' failed with error")
 
 
-def main():
-    host_dir = os.getcwd()
+def main(block_dir):
     container_dir = '/app'  # The directory inside the container where you want to mount
 
     commands = [
         "docker rm -f test_container",
```

Comment:
Do we want to use the name from the block spec itself?

Reply:
Yeah let's do that